DemoUI/index.html

```diff
@@ -187,12 +194,43 @@
           <i class='fa fa-angle-left backbutton' id='backup-detail-done-button'></i>
         </menu>
         <h1 id='detail-page-title'>EMPTY !!</h1>
-      </header>
+
+</header>
+
+
+      
 
       <section class='gaia-list fit scroll sticky' id='main-page' style='margin-top: 5rem; position: absolute'>
         <div data-type='list' style="overflow-y: scroll; height: calc(100% - 10rem); padding: 0">
-        </div>
+  
+
+  
+    
+  <table>
```

Comment:
A table can work here, but since we're in a list (from the div's `data-type` attribute above), we should probably be using list items like we are on the main page.